include/swift/AST/ModuleDependencies.h

```diff
@@ -311,6 +320,12 @@ class SwiftInterfaceModuleDependenciesStorage
   void updateCommandLine(const std::vector<std::string> &newCommandLine) {
     textualModuleDetails.buildCommandLine = newCommandLine;
   }
+
+  void addMacroDependency(StringRef module, StringRef libraryPath,
```

Comment:
Nit: `macroModuleName` would be more descriptive to disambiguate. Lots of "module"s in this file and type...